dataprofiler/profilers/graph_profiler.py

```diff
@@ -0,0 +1,363 @@
+            num_nodes=self._num_nodes,
+            num_edges=self._num_edges,
+            categorical_attributes=self._categorical_attributes,
+            continuous_attributes=self._continuous_attributes,
+            avg_node_degree=self._avg_node_degree,
+            global_max_component_size=self._global_max_component_size,
+            continuous_distribution=self._continuous_distribution,
+            categorical_distribution=self._categorical_distribution,
+            times=self.times,
+        )
+        return profile
+
+    def diff(self, other_profile, options=None):
+        """
+        Find the differences for two unstructured text profiles.
+
+        :param other_profile: profile to find the difference with
+        :type other_profile: GraphProfiler
+        :param options: options for diff output
+        :type options: dict
+        :return: the difference between profiles
+        :rtype: dict
+        """
+        cls = self.__class__
+        if not isinstance(other_profile, cls):
+            raise TypeError(
+                "Unsupported operand type(s) for diff: '{}' "
+                "and '{}'".format(cls.__name__, other_profile.__class__.__name__)
+            )
+        raise NotImplementedError("Function not yet implemented.")
+
+    def report(self, remove_disabled_flag=False):
+        """
+        Report on profile attribute of the class.
+
+        Pop value from self.profile if key not in self.__calculations
+        """
+        calcs_dict_keys = self.__calculations.keys()
+        profile = self.profile
+
+        if remove_disabled_flag:
+            profile_keys = list(profile.keys())
+            for profile_key in profile_keys:
+                # need to add props
+                if profile_key == "num_nodes":
+                    if "num_nodes" in calcs_dict_keys:
+                        continue
+                if profile_key == "num_edges":
+                    if "num_edges" in calcs_dict_keys:
+                        continue
+                if profile_key == "categorical_attributes":
+                    if "categorical_attributes" in calcs_dict_keys:
+                        continue
+                if profile_key == "continuous_attributes":
+                    if "continuous_attributes" in calcs_dict_keys:
+                        continue
+                if profile_key == "avg_node_degree":
+                    if "avg_node_degree" in calcs_dict_keys:
+                        continue
+                if profile_key == "global_max_component_size":
+                    if "global_max_component_size" in calcs_dict_keys:
+                        continue
+                if profile_key == "continuous_distribution":
+                    if "continuous_distribution" in calcs_dict_keys:
+                        continue
+                if profile_key == "categorical_distribution":
+                    if "categorical_distribution" in calcs_dict_keys:
+                        continue
```

Comment:
could we reduce this with a list and:
for profile_key in profile_keys:
                # need to add props
                if profile_key == ["num_nodes", ....]:
                    if profile_key in calcs_dict_keys:
                        continue